pkg/status/service_get_transitions.go

```diff
@@ -0,0 +1,62 @@
+package status
+
+import (
+	"errors"
+	"fmt"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/configuration"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/http/rest"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/service"
+	"net/http"
+)
+
+type ServiceGetTransitions struct {
+	issueId string
+}
+
+func (s *ServiceGetTransitions) InitJiraAPI(params configuration.JiraAPIResourceParameters) (rest.JiraAPI, error) {
+	s.issueId = params.ActiveIssue
+
+	return service.PreInitJiraAPI(s, params, http.MethodGet)
+}
+
+// See service/service.go for details
+func (s *ServiceGetTransitions) GetResults() map[string]string {
+	return nil
+}
+
+// See service/service.go for details
+func (s *ServiceGetTransitions) SetResultsFromPrevious(result map[string]string) {
+}
+
+// See service/service.go for details
+func (s *ServiceGetTransitions) GetEndpoint(url string) string {
+	return fmt.Sprintf("%s/issue/%s/transitions", url, s.issueId)
+}
+
+// See service/service.go for details
+func (s *ServiceGetTransitions) CreateRequestBody() []byte {
+	return nil
+}
+
+// See service/service.go for details
+func (s *ServiceGetTransitions) JSONResponseObject() interface{} {
+	return &Transitions{}
+}
+
+// See service/service.go for details
+func (s *ServiceGetTransitions) PostAPICall(result interface{}) error {
+	if transitions, ok := result.(*Transitions); !ok {
+		return errors.New("failed to convert result of type interface{} to issue of type reading.Issue")
+	} else {
+		TransitionsSlice = *transitions
+	}
+	return nil
+}
+
+func (s *ServiceGetTransitions) Name() string {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method ServiceGetTransitions.Name should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=5024349)